anaconda/Dockerfile

```diff
@@ -1,17 +1,18 @@
-FROM debian:8
-
-MAINTAINER Kamil Kwiek <kamil.kwiek@continuum.io>
+FROM debian:latest
 
 ENV LANG=C.UTF-8 LC_ALL=C.UTF-8
+ENV PATH /opt/conda/bin:$PATH
```

Comment:
Going to keep this set. There are situations when `~/.bashrc` isn't going to work.